include/deal.II/fe/fe_values.h

```diff
@@ -313,6 +369,24 @@ namespace FEValuesViews
                                  std::vector<typename ProductType<gradient_type,typename InputVector::value_type>::type> &gradients) const;
 
     /**
+     * Same as above, but using a of vector local degree-of-freedom values.
+     *
+     * The DoF values typically would be obtained in the following way:
```

Comment:
That works. There is also the possibility of adding a macro to `doc/doxygen/options.dox`.